### PR TITLE
refactored connectors

### PR DESCRIPTION
**Pull request by roeepolegfiverr**

### CODE REVIEW

File: shared/connectors.go [hunk 1]
```diff
@@ -4,12 +4,31 @@ import (
...
+	graylogPortKey     = "graylog.port"
+	graylogPortDefault = 9191
+)
+
 type clients struct {
-	Redis *redis.Client
-	Mongo *mgo.Session
+	Redis   *redis.Client
+	Mongo   *mgo.Session
+	Graylog *gelf.Gelf
```
Comment (nitsanfiverr) [inline shared/connectors.go]: Don't think Graylog should be here - it's not a connector. maybe we should put it somewhere else...
And I don't like the shared package name - we need to think of something else.

File: shared/connectors.go [hunk 2]
```diff
@@ -19,22 +38,36 @@ var (
 
 func init() {
 	Config = goenv.NewGoenv("./config/config.yml", goenv.GetEnv("GO_ENV", "development"), "")
-	var host, port, _ = Config.GetRedis()
-	local_redis, err := redis.Dial("tcp", fmt.Sprintf("%s:%s", host, port))
+
+	// Redis
+	redisHost := Config.Get(redisHostKey, redisHostDefault)
+	redisPort := Config.GetInt(redisPortKey, redisPortDefault)
```
Comment (nitsanfiverr) [inline shared/connectors.go]: We can write something nicer:
```
func (goenv *goenv.Goenv) OurRedis(redisName string) (host, port string) {
  host = goenv.Get(name+".host", "localhost")
	port = goenv.Get(name+".port", "6379")
return
}
```

File: shared/connectors.go [hunk 3]
```diff
@@ -19,22 +38,36 @@ var (
...
 		fmt.Println(err)
 	}
-	host = Config.Get("mongo.host", "localhost")
-	var i_port = Config.GetInt("mongo.port", 27017)
-	address := fmt.Sprintf("%s:%d", host, i_port)
+
+	// Mongo
+	mongoHost := Config.Get(mongoHostKey, mongoHostDefault)
+	mongoPort := Config.GetInt(mongoPortKey, mongoPortDefault)
+	address := fmt.Sprintf("%s:%d", mongoHost, mongoPort)
```
Comment (nitsanfiverr) [inline shared/connectors.go]: Same here -  as above

File: shared/connectors.go (hunk 1 above)
Comment (roeepolegfiverr) [inline shared/connectors.go]: It's a client. 
Agree on the shared package name. 
connectors might be better

**Pull request merged**